### ISSUE TITLE
[1.7.10] MASS SUGGESTION THREAD

### ISSUE THREAD

**Issue by Sunconure11** [ThexXTURBOXx/Expanded-Armory-Legacy#2]
I'm sure one can be created for newer versions of MC, but for now, I am going to list all ideas I have, since I was considering many of these for a TC4-bridging addon with several other mods. I will do more specialty things in my own mod, and let this mod handle the more generic stuff

MASSIVE LIST WARNING

- [ ] Neptunium [From aquaculture]
- [ ] Kreknorite [From falling meteors]
- [ ] Frezarite [From falling meteors]
- [ ] Meteorite [From falling meteors]
- [ ] Silver [a fairly common metal, but some special witchery compat could be done. Get in touch with the dev of Deathly Hallows on how they made a Witchery addon]
- [ ] Scarab [F&A/Fossils and Archaeology]
- [ ] Various materials from Atum [Atum]
- [ ] Nebula [Advanced Botany]
- [ ] Jade [Erebus]
- [ ] Various Aether materials [Various builds of Aether]
- [ ] Bound weapons [Blood Magic]
- [ ] Various materials from Pam's Mods [Pam's various mods that aren't Harvestcraft]
- [ ] Various Tropicraft materials [Tropicraft]
- [ ] Metallurgy metals [Metallurgy]
- [ ] Mo' Creatures materials [Mo' Creatures]
- [ ] Et Futurum's Netherite [Et Futurum]
- [ ] AM2Playground [AM2 Playground]
- [ ] Twilight Forest materials [Twilight Forest]
- [ ] Natura Woods [Natura]
- [ ] Runic Steel [Runic Dungeons]
- [ ] Gany's Mods [Gany's various improvements for the nether, surface, and end]
- [ ] Thaumic Bases materials [Thaumic Bases]
- [ ] Aura Cascade materials [Aura Cascade]
- [ ] Cardboard {Better Storage]
- [ ] Bound weapons [Ars Magica 2]
- [ ] Project Red gems [Project Red]
- [ ] Blood Infused materials [Blood Arsenal]
- [ ] Amethyst [Biomes o' Plenty]
- [ ] Mud [Biomes o' Plenty]
- [ ] Spectral [Electroblob's Wizardry]
- [ ] Adaminite [Thaumic Additions]
- [ ] Netherlicious materials [Netherlicious]
- [ ] Dark Matter [ProjectE]
- [ ] Red Matter [ProjectE]
- [ ] Various gems [Gems & Jewels]
- [ ] Various addons for Simple Ores (https://modrinth.com/user/Sinhika)
- [ ] The Betweenlands
- [ ] Ice and Fire

Also, if this is pursued, check this out for info: https://legacymoddingmc.github.io/wiki/

If you can think of other mods, let me know.

**Comment by ThexXTURBOXx** [comment]
I have to admit that it is very tempting to add more material support directly from within BWM: Legacy, but I think it is more beneficial to keep doing what was done in the original mod: keeping it "separate" (actually, I think that BalkondeurAlpha had no intentions to add mod integration at all, but I don't know for certain).
My compromise would be the following: Also reviving Expanded Armory as "Expanded Armory: Legacy" or something like that.
For that purpose, I have already [forked Expanded Armory as-is](https://github.com/ThexXTURBOXx/Expanded-Armory-Legacy) (in case the owner decides to delete it in the mean time - which is unlikely, but still a possibility).
I will focus on BWM: Legacy first, anyway!

But thank you very much for the huge list already! I have also added two ;)

**Comment by ThexXTURBOXx** [comment]
I think I will move this issue :)

**Comment by Sunconure11** [comment]
@ThexXTURBOXx BTW, the last time I remember working with you, which was when I was waaaaaaaay younger, I believe you mentioned you had some form of color-blindness, and it showed in some of the work done, so I had to color-correct textures. I suggest that when you tackle this, to hunt down an artist to help with this. Because there are a lot of potential mods you could hook into, for starters.

**Comment by ThexXTURBOXx** [comment]
Yes, indeed, I have Deuteranopia and Protanopia. As soon as I want to hook into more mods than currently, I will try to find someone, who wants to take this over! Of course, if you want, you can still be the one, but I completely understand if you don't have the time currently!

**Comment by Sunconure11** [comment]
As I had written a lot, I would need someone else's aid lol.

As for some of the stuff, like the weapons that bridge certain magic mods together, would it be possible to add them into said books? I.e. entries on manasteel weaponry in Botania, entries on thaumium weapons in Thaumcraft, etc

**Comment by Sunconure11** [comment]
If you have Discord, ask around in the various modding Discords for aid when the time comes.

**Comment by ThexXTURBOXx** [comment]
I remember that I had actually accomplished that in Reforged: Integration, so it is indeed possible to do that :)
And yes, I do have Discord (and I made an own one a couple of weeks ago, but I am too scared to share it yet. There are too many trolls out there and I don't have anyone who could moderate it besides me)

**Comment by Sunconure11** [comment]
@ThexXTURBOXx Anything new here?

**Comment by ThexXTURBOXx** [comment]
Not yet - I was away from my home for the last three weeks and couldn't work on either BWM or EA. I will be home rather soon and will take a look at the newly reported bugs over at BWM first and then focus on EA

**Comment by 167OSS** [comment]
I am looking forward to the progress of this project.

**Comment by Sunconure11** [comment]
> Yes, indeed, I have Deuteranopia and Protanopia. As soon as I want to hook into more mods than currently, I will try to find someone, who wants to take this over! Of course, if you want, you can still be the one, but I completely understand if you don't have the time currently!

On the note of your colorblindness, I had asked on Reddit about this a month ago or so.

https://www.reddit.com/r/ColorBlind/comments/1e7hx30/what_would_the_world_look_like_to_an_individual/

Someone suggested you have blue cone monochromacy. I am not sure how that is tested for, but you may want to look into it.

**Comment by ThexXTURBOXx** [comment]
Oh nice - I had never thought about asking on Reddit whether there are some people that can relate, so that's very nice to see :D

> Someone suggested you have blue cone monochromacy. I am not sure how that is tested for, but you may want to look into it.

It's actually very hard to test... <completely off-topic philosophical stuff>
If you think about it: How would you describe a colour like red to someone who may or may not see it the same way as you?
I mean, you could also try describing the colour red to a person who is blind (even though, that might be even harder since you would need to somehow describe vision itself before that).
Most of the time, I get asked "Which colour is this pen? Which colour is this [...]?" and I am like "I don't know - probably red, probably brown, probably green" (Important to note: Nearly everyone who ever asked me that thought that I would see everything in white/gray/black - this is definitely not true. I can say with 100% certainty that what people tell me is gray looks different to me than things that people tell me are either red, green, brown, or even other colours. So, I do see saturation, still).
Most of the time I am guessing based on what I've learned before - I know that leaves are generally green, but turn brown in autumn.
From that, I get some context clues on what colour certain things have.
For instance, because I was told yesterday that the sky was blue, it doesn't make sense that the sky would be purple today (bad example, but I can't come up with a better one at the moment). But if it really is purple today, I would not notice.
One thing I learned to use is brightness and (to some degree) saturation to figure out what colour something could be, but also that is rather unspecific since both properties are perfectly orthogonal to hue.
However: apparently, some shades of colours are not really used in the real world and hence, I can sort of "exclude" them when I want to identify some colour.
Pretty hard to describe my thought process when trying to identify colours : ^)

But in a certain sense, I am quite lucky to be alive nowadays and not a century ago: I have an app on my smartphone "Color picker", which allows me to take a picture of something and it tells me what colour it has.
And even better than that, there are these EnChroma glasses, which apparently let you see colours (by shifting or intensifying them or something like that). Didn't test them myself, though. But their online test was quite positive that they may work for me... Well, maybe one day!
And lastly, AIs are also improving and they will certainly play a role in colour recognition/description at some point in the near future.
</completely off-topic philosophical stuff>

Oh, and maybe one last thing: At least here in Germany, doctors only rarely say that someone has a colour vision deficiency (CVD), but that is probably what I have.
Like, not really completely red- and green-blindness, but rather both red- and green-vision deficiencies simultaneously.
Basically, CVD is synonymous to colorblindness for most people here.

**Comment by ThexXTURBOXx** [comment]
By the way, I don't want to keep you in the dark, obviously: I am working on this behind closed curtains currently. Just like I did with BWM: Legacy.
This is because it is hugely unfinished and there may be bugs here and there, which I need to find and fix, accordingly.
Probably won't happen, but I don't want someone build this mod from source and then complain to me why X and Y is not working correctly and so on.
I will upload the finished product as soon as it is ready, though, obviously!

**Comment by Sunconure11** [comment]
While I haven't played MC in a bit, I saw this while looking for something:

https://www.curseforge.com/minecraft/mc-mods/chitin-equipment

**Comment by ThexXTURBOXx** [comment]
Good news: I have just published version 1.5.0 - only for 1.7.10 yet, sadly.
When I find the time, I will continue the updates to newer versions and also implement more materials!

If you want to help translating this mod, feel free to do so here: https://crowdin.com/project/balkons-weaponmod-legacy

**Comment by PopcornHead51** [comment]
I started a similar project about a year ago but didn't get much done. If you want sprites for the jade weapons from Erebus lmk. I tried to keep the style similar to the other jade weapons/tools in the Erebus mod.

**Comment by PopcornHead51** [comment]
Also, could you add the Betweenlands' ore to the list?

**Comment by ThexXTURBOXx** [comment]
> If you want sprites for the jade weapons from Erebus lmk. I tried to keep the style similar to the other jade weapons/tools in the Erebus mod.

Of course! If you already have them ready, I would greatly appreciate them and give you the credit for those, obviously! :)

> Also, could you add the Betweenlands' ore to the list?

Added, thanks for the suggestion!

**Comment by PopcornHead51** [comment]
> > If you want sprites for the jade weapons from Erebus lmk. I tried to keep the style similar to the other jade weapons/tools in the Erebus mod.
> 
> Of course! If you already have them ready, I would greatly appreciate them and give you the credit for those, obviously! :)
> 
> > Also, could you add the Betweenlands' ore to the list?
> 
> Added, thanks for the suggestion!

Here's the sprites I made. Also, you should check out https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/wip-mods/2315124-compatibility-mod-for-balkons-weapon-mod-and. It has many of the items for compat with Metallurgy done. The art is good but it was coded fairly poorly--there's some bugs and its fairly crash-heavy. Not sure how you'd get perms though since the author seems to have been inactive for the past few years.
[items.zip](https://github.com/user-attachments/files/18894672/items.zip)

**Comment by ThexXTURBOXx** [comment]
Ah yes, I do remember that mod - it's even [archived in my MCModArchive](https://mcmodarchive.femtopedia.de/#Mods/Z/zombiebear%20bwm%20metallurgy%20compatibility/1.7.10/) (btw: if you have either 0.1.0 or 0.1.1 by chance, feel free to let me know!)
Thanks for the sprites - I will see when I have time to incorporate them!

**Comment by Omgise** [comment]
@ThexXTURBOXx Allow diamond weapons to be upgraded to netherite weapons using the Et Futurum Requiem Smithing Table.
Then, add a configuration option to disable netherite weapon crafting recipes, allowing netherite weapons to be obtained only by upgrading diamond weapons at the Smithing Table.
